### PR TITLE
Grammatical fixes for Adapter

### PR DESCRIPTION
**Pull request by iluwatar**
Fix some grammatical issues in the Adapter pattern.

### CODE REVIEW

File: adapter/README.md [hunk 1]
```diff
@@ -100,25 +100,25 @@ captain.row();
...
 
-* you want to use an existing class, and its interface does not match the one you need
-* you want to create a reusable class that cooperates with unrelated or unforeseen classes, that is, classes that don't necessarily have compatible interfaces
-* you need to use several existing subclasses, but it's impractical to adapt their interface by subclassing every one. An object adapter can adapt the interface of its parent class.
-* most of the applications using third party libraries use adapters as a middle layer between the application and the 3rd party library to decouple the application from the library. If another library has to be used only an adapter for the new library is required without having to change the application code.
+* You want to use an existing class, and its interface does not match the one you need
+* You want to create a reusable class that cooperates with unrelated or unforeseen classes, that is, classes that don't necessarily have compatible interfaces
+* You need to use several existing subclasses, but it's impractical to adapt their interface by subclassing every one. An object adapter can adapt the interface of its parent class.
```
Comment (ohbus) [inline adapter/README.md:105]:
```suggestion
* You need to use several existing subclasses, but it's impractical to adapt their interface by subclassing everyone. An object adapter can adapt the interface of its parent class.
```

File: adapter/README.md [hunk 2]
```diff
@@ -100,25 +100,25 @@ captain.row();
...
+* Most of the applications using third-party libraries use adapters as a middle layer between the application and the 3rd party library to decouple the application from the library. If another library has to be used only an adapter for the new library is required without having to change the application code.
 
 ## Consequences:
 Class and object adapters have different trade-offs. A class adapter
 
-*	adapts Adaptee to Target by committing to a concrete Adaptee class. As a consequence, a class adapter won’t work when we want to adapt a class and all its subclasses.
-*	let’s Adapter override some of Adaptee’s behavior, since Adapter is a subclass of Adaptee.
-*	introduces only one object, and no additional pointer indirection is needed to get to the adaptee.
+*	Adapts Adaptee to Target by committing to a concrete Adaptee class. As a consequence, a class adapter won’t work when we want to adapt a class and all its subclasses.
```
Comment (ohbus) [inline adapter/README.md:111]: Will this be `adopt`?

Or should we stick to something like `change` or `modify`?

```suggestion
*	Adapts Adaptee to Target by committing to a concrete Adaptee class. As a consequence, a class adapter won’t work when we want to adopt a class and all its subclasses.
```

**Review by ohbus — COMMENTED** [review]
Just a discussion

File: adapter/README.md (hunk 2 above)
Comment (ohbus) [reply adapter/README.md:111]: @iluwatar any thoughts on this?

Comment (iluwatar) [reply adapter/README.md:111]: > @iluwatar any thoughts on this?

My original thought was to use the verb `adapt` as in the adapter.

**Review by ohbus — APPROVED** [review]

**Comment by ohbus** [comment]
Thank you @iluwatar this has been long pending.

**Pull request merged**